README.md

```diff
@@ -59,13 +59,15 @@ Once all agent instances are registered, the service owner re-deploys the servic
 The implementation of such multisig is provided here:
 - [GnosisSafeSameAddressMultisig](https://github.com/valory-xyz/autonolas-registries/blob/main/contracts/multisigs/GnosisSafeSameAddressMultisig.sol)
 
+To verify the multisig data, use the hardhat script along with the helper JSON file located [here](https://github.com/valory-xyz/autonolas-registries/blob/main/scripts/multisig/)
```

Comment:
Instead of adding this here, my suggestion is to have a README in the script/multisig foldes with instructions. Similiral to what we have for script/deployment folder. So that, anyone can go and test or update the multisig info with the data required.

Reply:
Please provide which info has to be put where, or just modify the PR as you see fits.

Reply:
The following information would be necessary, let me know if you can add those.
- what it is necessary to add/modify in the multising_info.json
- how to run the script: simply saying that once exported local variable the following is necessary`npx hardhat run - scripts/multisig/script-name"
- how to use the script results

Reply:
Please write these in a README you would like to add and push directly into this PR.